src/components/atoms/picker-sheet/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Picker } from 'react-native';
 
 // React Native's own picker is deprecated, this is recommended and supported in expo SDK 38
 // Import from @react-native-community/picker/js/Picker to avoid warnings due to
 // https://github.com/react-native-community/react-native-picker/issues/109
-import Picker from '@react-native-community/picker/js/Picker';
+// remove because storybook does not work with it
+// https://github.com/react-native-community/react-native-picker/issues/45#issuecomment-633163973
+// import Picker from '@react-native-community/picker/js/Picker';
```

Comment:
@AlanSl can you please look at this, seems like you used `@react-native-community/picker/js/Picker` that does not work with storybook and its not recommended for expo as well `https://github.com/react-native-community/react-native-picker/issues/45#issuecomment-633163973` 

I replace it with regular RN picker and it works fine, at least for now.

Reply:
It's Expo SDK 37 that `@react-native-community/picker` doesn't work with. We're on Expo SDK 38 which does work with it. (Also, [I'd already commented](https://github.com/react-native-community/react-native-picker/issues/45#issuecomment-654327582) on that thread you linked to 😊).

`react-native/picker` is [officially deprecated](https://reactnative.dev/docs/picker) and gives this yellow box warning:

> Picker has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/picker' instead of 'react-native'. See https://github.com/react-native-community/react-native-picker

So we should avoid using it if we can. It also has some weird display issues on Android that the non-deprecated one doesn't have.

What's the problem with Storybook? Do you still get the error if you run `expo update`?